package.json

```
@@ -9,7 +9,12 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "gulp lint"
+    "lint": "yarn lint:schemas && yarn lint:test-schemas",
+    "lint:schemas": "prettier --check \"schemas/**/*.json\" && eslint \"schemas/**/*.json\"",
+    "lint:test-schemas": "prettier --check \"tests/schemas/*/*.json\" && eslint \"tests/schemas/**/*.json\"",
+    "format": "yarn format:schemas && yarn format:test-schemas",
+    "format:schemas": "prettier \"schemas/*/*.json\" --write && eslint --fix \"schemas/*/*.json\"",
+    "format:test-schemas": "prettier \"tests/schemas/*/*.json\" --write && eslint --fix \"tests/schemas/*/*.json\""
```

The globs need to be updated to supports nested directory that aren't just at the top level. At the moment, not all files are picked up. 

Also, we seem to be not linting and formatting the `ajv` folder, but we should be. `AJV` folder fails lint atm and should be fixed.
```suggestion
    "lint": "yarn lint:schemas && yarn lint:test-schemas && yarn lint:ajv",
    "lint:schemas": "prettier --check \"schemas/**/*.json\" && eslint \"schemas/**/*.json\"",
    "lint:test-schemas": "prettier --check \"tests/schemas/**/*.json\" && eslint \"tests/schemas/**/*.json\"",
    "lint:ajv": "prettier \"ajv/**/*.js\" && eslint \"ajv/**/*.js\"",
    "format": "yarn format:schemas && yarn format:test-schemas && yarn format:ajv",
    "format:schemas": "prettier \"schemas/**/*.json\" --write && eslint --fix \"schemas/**/*.json\"",
    "format:test-schemas": "prettier \"tests/schemas/**/*.json\" --write && eslint --fix \"tests/schemas/**/*.json\"",
    "format:ajv": "prettier \"ajv/**/*.js\" --write && eslint --fix \"ajv/**/*.js\""
```

---

Well that broke :laughing: